lib/tasks/myturn_import.rake

```diff
@@ -0,0 +1,136 @@
+# For reference, these are the columns in a complete export (shown in alphabetical order):
+#
+# "Additional Image"
+# "Admin Notes"
+# "Agreement that must be signed to checkout the item"
+# "Amount / Fee"
+# "Attachment"
+# "Author"
+# "Between Buffer Days"
+# "Categories"
+# "Color"
+# "Condition"
+# "Current Location"
+# "Daily Late Fee"
+# "Date Created"
+# "Date Last Edited"
+# "Date Last Updated"
+# "Date Purchased"
+# "Default Loan Length"
+# "Default Scheduled Maintenance Plan"
+# "Description"
+# "Dimensions (WxHxD)"
+# "Eco Rating"
+# "Embodied Carbon"
+# "Emission Factor"
+# "Excluded Fulfillment Methods"
+# "Featured"
+# "Floating"
+# "Goes Home"
+# "Grace period on late fees (in days)"
+# "Historical Cost"
+# "Home Location"
+# "Image"
+# "Item ID"
+# "Item Type"
+# "Keywords"
+# "Location Code"
+# "Maintenance Plan After Each Check In"
+# "Manufacturer"
+# "Max Frequency Units"
+# "Max Frequency"
+# "Max Output"
+# "Maximum number of renewals"
+# "Maximum number that may be reserved"
+# "Maximum percentage of inventory that may be reserved"
+# "Maximum Reservation Length"
+# "Min Frequency Units"
+# "Min Frequency"
+# "Model"
+# "Name"
+# "Now Buffer Days"
+# "Post-Description Text"
+# "Pre-Description Text"
+# "Product Code / UPC"
+# "Publisher"
+# "Purchased"
+# "Replacement Cost"
+# "Serial Number"
+# "Size"
+# "Source / Supplier"
+# "Status(es)"
+# "Tax(es)"
+# "Weight"
+
+namespace :myturn do
+  task reset: :environment do
+    # Just for development
```

Comment:
Might want to through a `raise unless Rails.env.development?` at the top of this task.

Reply:
Good call!